examples/functions-with-included-record-parameters/functions_with_included_record_parameters.bal

```diff
@@ -0,0 +1,83 @@
+import ballerina/io;
+ 
+type Student record {
```

Comment:
Shall we make this also a closed record?

Reply:
Fixed.